static/app/views/discover/table/tableView.tsx

```diff
@@ -190,7 +190,10 @@ function TableView(props: TableViewProps) {
 
       let target;
 
-      if (dataRow.trace === null || dataRow.trace === undefined) {
+      if (
+        !dataRow.trace ||
+        ['error', 'default'].includes(dataRow['event.type'].toString())
```

Comment:
I think this should be `dataRow['event.type'].toString() != 'transaction'` as we have other error event types such as csp and a few more that I don't recall.

Reply:
Ahh I se